treeherder/etl/perf_data_adapters.py

```diff
@@ -91,13 +92,25 @@ def _round(num):
 
     @staticmethod
     def _calculate_summary_data(job_id, result_set_id, push_timestamp, results):
-        values = []
+        # to calculate the summary geometric mean, we drop the first 5
+        # replicates from a performance series, and take the median of the
+        # rest
+        NUM_REPLICATES_TO_DROP = 5
```

Comment:
on android only svg has 7 replicates, tp4m is 3 and tcheck is 5.  This won't help as much.